stress-tests/graph/GomoryHu.cpp

```diff
@@ -0,0 +1,46 @@
+#include "../utilities/template.h"
+#include "../utilities/utils.h"
+
+#include "../../content/graph/GomoryHu.h"
+#include "../../content/graph/Dinic.h"
+
+
+void test(int N, int mxFlow, int iters) {
+    for (int it = 0; it < iters; it++) {
```

Comment:
set `n = rand() % N` or `n = rand() % N + 1` here and use that n instead to improve test coverage